src/hexo/themes/documentation/source/images/results-pwa.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 72 72" style="enable-background:new 0 0 72 72;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#ACADDA;}
+	.st1{fill:#2D28A1;}
+</style>
+<title>icons</title>
+<g>
+	<path class="st0" d="M32.8,59.3h-19C6.2,59.3,0,53.1,0,45.5v-19c0-7.6,6.2-13.8,13.8-13.8h19c7.6,0,13.8,6.2,13.8,13.8v19
+		C46.6,53.1,40.5,59.3,32.8,59.3z M13.8,17c-5.2,0-9.5,4.3-9.5,9.5v19c0,5.2,4.3,9.5,9.5,9.5h19c5.2,0,9.5-4.3,9.5-9.5v-19
+		c0-5.2-4.3-9.5-9.5-9.5H13.8z"/>
+	<path class="st1" d="M16.7,33.4c0.5,0.4,0.7,1,0.6,1.7c0,0.7-0.2,1.3-0.7,1.8c-0.5,0.4-1.2,0.7-1.9,0.6h-1.1v1.8h-1.7v-6.4h2.8
+		C15.4,32.8,16.1,33,16.7,33.4z M15.5,35.8c0.2-0.2,0.3-0.5,0.3-0.7c0-0.3-0.1-0.5-0.3-0.7c-0.2-0.2-0.5-0.3-0.8-0.2h-1.1v2h1.1
+		C15,36.1,15.2,36,15.5,35.8z M27.9,32.8l-2.2,6.4H24l-1.2-4.4l-1.2,4.4h-1.7l-2.2-6.4h1.8l1.3,4.7l1.3-4.7h1.6l1.3,4.7l1.2-4.7
+		H27.9z M32.5,38.1h-2.8l-0.5,1.1h-1.7l2.8-6.4H32l2.7,6.4h-1.8L32.5,38.1z M32,36.8l-0.9-2.3l-0.9,2.3H32z"/>
+</g>
+</svg>
```

Comment:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 72 72"><path d="M32.8 59.3h-19C6.2 59.3 0 53.1 0 45.5v-19c0-7.6 6.2-13.8 13.8-13.8h19c7.6 0 13.8 6.2 13.8 13.8v19c0 7.6-6.1 13.8-13.8 13.8zM13.8 17c-5.2 0-9.5 4.3-9.5 9.5v19c0 5.2 4.3 9.5 9.5 9.5h19c5.2 0 9.5-4.3 9.5-9.5v-19c0-5.2-4.3-9.5-9.5-9.5h-19z" fill="#acadda"/><path d="M16.7 33.4c.5.4.7 1 .6 1.7 0 .7-.2 1.3-.7 1.8-.5.4-1.2.7-1.9.6h-1.1v1.8h-1.7v-6.4h2.8c.7-.1 1.4.1 2 .5zm-1.2 2.4c.2-.2.3-.5.3-.7 0-.3-.1-.5-.3-.7-.2-.2-.5-.3-.8-.2h-1.1v2h1.1c.3-.1.5-.2.8-.4zm12.4-3l-2.2 6.4H24l-1.2-4.4-1.2 4.4h-1.7l-2.2-6.4h1.8l1.3 4.7 1.3-4.7h1.6l1.3 4.7 1.2-4.7h1.7zm4.6 5.3h-2.8l-.5 1.1h-1.7l2.8-6.4H32l2.7 6.4h-1.8l-.4-1.1zm-.5-1.3l-.9-2.3-.9 2.3H32z" fill="#2d28a1"/></svg>
```

Reply:
@qzhou1607 can you update this file with the content of @alrra's comment?